.changelog/524.cfg.md

```diff
@@ -0,0 +1 @@
+Update routes to conform to EIP-3091
```

Comment:
Please, add a link to EIP-3091 standard like:
```
Update routes to conform to [EIP-3091]

[EIP-3091]: https://eips.ethereum.org/EIPS/eip-3091
```

Reply:
And classify it as a breaking change

Reply:
Added link, reanamed to breaking.